addons/download-button/addon.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Project page download button",
+  "description": "Allows you to download projects directly from the project page, without having to open the editor first. The Download button is added next to the Copy Link button.",
+  "tags": ["community", "projectPlayer", "recommended"],
```

Recommended? Probably not...

---

Probably should be `featured`.
```suggestion
  "tags": ["community", "projectPlayer", "featured"],
```